install.sh

```diff
@@ -35,34 +35,66 @@ DOWNLOAD_URL="https://github.com/kubescape/kubescape/releases/${RELEASE}/kubesca
 
 curl --progress-bar -L $DOWNLOAD_URL -o $OUTPUT
 
-# Checking if SUDO needed/exists 
+# Find install dir
+install_dir=/usr/local/bin # default if running as root
+if [ "$(id -u)" -ne 0 ]; then
+  install_dir=$BASE_DIR/bin # if not running as root, install to user dir
+  export PATH=$PATH:$BASE_DIR/bin
+fi
+
+# Create install dir if it does not exist
+if [ ! -d "$install_dir" ]; then
+  mkdir -p $install_dir
+fi
+
+chmod +x $OUTPUT 2>/dev/null
+
+# cleaning up old install
 SUDO=
-if [ "$(id -u)" -ne 0 ] && [ -n "$(which sudo)" ]; then
+if [ "$(id -u)" -ne 0 ] && [ -n "$(which sudo)" ] && [ "$KUBESCAPE_EXEC" != "" ] && [ -f /usr/local/bin/$KUBESCAPE_EXEC ]; then
     SUDO=sudo
+    echo -e "\n\033[33mOld installation as root found, do you want to remove it? [\033[0my\033[33m/n]:"
+    read -n 1 -r
+    if [[ ! $REPLY =~ ^[Yy]$ ]] && [[ "$REPLY" != "" ]]; then
+        echo -e "\n\033[0mSkipping old installation as root removal."
+    else
+        echo -e "\n\033[0mWe will need the root access to uninstall the old kubescape CLI."
+        if $SUDO rm -f /usr/local/bin/$KUBESCAPE_EXEC 2>/dev/null; then
+            echo -e "\033[32mRemoved old installation as root at /usr/local/bin/$KUBESCAPE_EXEC"
+        else
+            echo -e "\033[31mFailed to remove old installation as root at /usr/local/bin/$KUBESCAPE_EXEC, please remove it manually."
+        fi
+    fi
 fi
 
+if [ "$KUBESCAPE_EXEC" != "" ]; then
+    if [ "${SUDO_USER:-$USER}" != "" ]; then
+        rm -f /home/${SUDO_USER:-$USER}/.kubescape/bin/$KUBESCAPE_EXEC 2>/dev/null || true
```

Comment:
SC2086: Double quote to prevent globbing and word splitting.
```suggestion
        rm -f /home/"${SUDO_USER:-$USER}"/.kubescape/bin/$KUBESCAPE_EXEC 2>/dev/null || true
```